common/src/test/java/net/opentsdb/collections/TestDirectByteArray.java

```diff
@@ -0,0 +1,266 @@
+// This file is part of OpenTSDB.
+// Copyright (C) 2021  The OpenTSDB Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package net.opentsdb.collections;
+
+import java.util.Random;
+
+public class TestDirectByteArray {
+//  @Test
```

Comment:
why are the tests commented?

Reply:
Cause TSD is still using junit 4x. I need to move to 5x